python/hsfs/core/feature_view_engine.py

```diff
@@ -967,39 +970,46 @@ def get_feature_logging(self, fv):
         )
 
     def _get_logging_fg(self, fv, transformed):
-        feature_logging = self.get_feature_logging(fv)
-        if transformed:
-            return feature_logging.transformed_features
-        else:
-            return feature_logging.untransformed_features
+        return self.get_feature_logging(fv).get_feature_group(transformed)
 
     def log_features(
         self,
-        fv,
-        features,
-        prediction=None,
-        transformed=False,
-        write_options=None,
-        training_dataset_version=None,
+        fv: FeatureView,
+        feature_logging: FeatureLogging,
+        features_rows: Union[
+            pd.DataFrame, list[list], np.ndarray, TypeVar("pyspark.sql.DataFrame")
+        ],
+        predictions: Optional[Union[pd.DataFrame, list[list], np.ndarray]] = None,
+        transformed: Optional[bool] = False,
+        write_options: Optional[Dict[str, Any]] = None,
+        training_dataset_version: Optional[int] = None,
         hsml_model=None,
```

Comment:
Can you please change the name of `hsml_model` here as well? so we keep the same name across methods.

Reply:
I think we can keep the internal method as `hsml_model` so to make the variable more understandable